resources/core-overwrites/layouts/checkout/footer.blade.php

```diff
@@ -0,0 +1,9 @@
+<footer class="mt-auto">
+    <div class="mt-10 flex items-center justify-center gap-6 border-b-4 border-b-ct-enhanced bg-ct-inactive-100 py-4 text-sm text-ct-inactive">
+        @foreach (['Privacy Policy', 'Terms & Conditions', 'About us'] as $link)
+            <a class="hover:underline" href="#">
+                {{ $link }}
```

Comment:
Do we care about translations for placeholder data like this?

Reply:
Not for me we always override this file to get the data from a CMS.

Reply:
No, this is the same as account features. they're static and should get replaced in the project itself with relevant data

Reply:
I thought we used lang tags instead of such foreach because of translation tools not being able to scan these type of solutions? @Jade-GG